test-runner-poc/environments.js

```diff
@@ -135,11 +140,8 @@ class ShardedEnvironment extends EnvironmentBase {
   setup(callback) {
     const shardOptions = this.options && this.options.shard ? this.options.shard : {};
     // First set of nodes
-    const nodes1 = [
-      genShardedConfig(31010, { tags: { loc: 'ny' } }, shardOptions),
-      genShardedConfig(31011, { tags: { loc: 'sf' } }, shardOptions),
-      genShardedConfig(31012, { arbiter: true }, shardOptions)
```

Comment:
I think we'll need to do some investigation into whether an `arbiter` is important to have.

Reply:
We currently are not adding an `arbiter` inside the `generateNodesArray` function